docs/_docs/ssl-tls.adoc

```diff
@@ -0,0 +1,193 @@
+= SSL/TLS
+
+This page explains how to configure SSL/TLS encryption between the cluster nodes (server and client) and the clients that connect to your cluster.
+
+== Considerations
+
+All internal connections in the cluster context, as well as cluster's user interaction interfaces, are SSL-enabled. The communication categories are as follows: 
+
+* Between the user and the cluster (node): REST
+* Between the user and the platform clients:
+
+** CLI
+** Client
+
+* Between nodes: Network (Messaging, Scalecube)
+
+All SSL configurations activities are performed at the node level.
+
+Due to Micronaut limitations, Apache Ignite does nopt support direct paths to SSL certificates. Instead, it utilizes PKCS12 and JKS keystore.
+//Need a link to Micronaut in general; if possible, to the limitation in particular 
+
+== REST
+
+The standard implementation of SSL support for REST involves configuring a secure connection on a separate port. Apache Ignite also supports the DUAL protocol feature.
+//Need a link to a description of the DUAL protocol. 
```

Comment:
I think there is no standard specification for dual protocol. I would say it will be enough to explain that dual protocol means both HTTP and HTTPS are supported on different ports.